postgres/init.sls

```diff
@@ -15,7 +15,7 @@ install-postgresql-repo:
 install-postgresql:
   pkg.installed:
     - name: {{ postgres.pkg }}
-    - refresh: {{ 'use_upstream_repo' in pillar.get('postgres') }}
+    - refresh: {{ salt['pillar.get']('postgres:use_upstream_repo') }}
```

Comment:
I think you actually want to set a default here, otherwise refresh will get set to exceptions.KeyError.
```
{{ salt['pillar.get']('postgres:use_upstream_repo', False) }}
```